### ISSUE TITLE
valid tags in langtags return null

### ISSUE THREAD

**Issue by DavidLRowe** [silnrsi/langfontfinder#18]
GT reports: 
I ran the mui-language-picker deployment script and it identified the following tags are missing at [lff.api.languagetechnology.org/lang](http://lff.api.languagetechnology.org/lang):
Request failed with status code 404: bap-Krai
Request failed with status code 404: elx
Request failed with status code 404: gvr-Gukh
Request failed with status code 404: kru-Tols
Request failed with status code 404: kyw-Chis
Request failed with status code 404: suz-Sunu
Request failed with status code 404: tcy-Tutg
Request failed with status code 404: tyj-Tayo
Request failed with status code 404: unr-Onao
Request failed with status code 404: wo-Gara

Also: using the und language code to look up the Script tag also doesn't work for these scripts (with elx being the Pelm script).

**Comment by DavidLRowe** [comment]
IIUC you're expecting that any tag from langtags should return something from the LFF API, even if it is only an empty list. Is that right?

**Comment by LornaSIL** [comment]
So, I think `script2font.csv` needs updating to the newest ISO 15924 code additions.

The `Pelm` one is misspelled in our table and we have a double-entry for `Perm` which is actually Old Permic. 

I don't think there will be fonts yet since all by `Pelm` are going into Unicode 16.0 (Sept 2024), but I'm happy to update this file with what is available, BUT is this file the authoritative source or is it derived from somewhere else?

**Comment by DavidLRowe** [comment]
Yes, updates to `script2font.csv` should be made in this repository. Thanks!

**Comment by LornaSIL** [comment]
I think this is fixed now. Now the error is `LDML SERVER ERROR: No file was found to match your request: elx.` which probably means there is no sldr file for it.

**Comment by DavidLRowe** [comment]
@gtryus Lorna updated the list of scripts and fixed a typo. I don't think the LFF service will return a font for some of these scripts, since there are no fonts listed in the .csv file.

**Comment by imnasnainaec** [comment]
Now running the MLP scripts, the only one missing is "suz-Sunu" (https://github.com/sillsdev/mui-language-picker/issues/28).

**Comment by LornaSIL** [comment]
I don't know of any Unicode fonts for Sunuwar, do you?

**Comment by imnasnainaec** [comment]
@LornaSIL It looks like Jamra Patel developed one for The Sunuwar Welfare Society of Nepal (https://r12a.github.io/scripts/sunu/suz.html#acknowledgements, https://www.jamra-patel.com/commissions-pro-bono) called Mukdum (unofficially available here: https://languagetools-153419.appspot.com/suz/downloads/). According to Patel's conversion tool (https://sites.google.com/jamra-patel.com/sunuwar-conversion/sunuwar-encoding-conversion), he made a "Mukdum1-Uni-Prelim.ttf" Unicode font, but I can't find that for download.

EDIT: The appspot link does offer "Mukdum Final Unicode TTF" and "Mukdum Unicode TTF". I haven't investigated their legitimacy, but they can be experimented with on the parent page: https://languagetools-153419.appspot.com/suz/.

**Comment by imnasnainaec** [comment]
(It looks like this issue can be closed.)

**Comment by LornaSIL** [comment]
Mukdum.ttf has no license or copyright. I don't think we would be willing to use that.

**Comment by DavidLRowe** [comment]
@LornaSIL In a [CLDR PR](https://github.com/unicode-org/cldr/pull/4651/files), Steven Loomis is using SEISunuwar-Regular.ttf but I don't know the source or status of that font.

**Comment by LornaSIL** [comment]
That font (SEISunuwar-Regular.ttf) does indicate it's under the OFL. But, I can't seem to find it anywhere in a public place.

**Comment by n7s** [comment]
Looks like upstream is here: https://github.com/ScriptEncodingInitiative/sunuwar

**Comment by n7s** [comment]
to be more specific in the [sei folder](https://github.com/ScriptEncodingInitiative/sunuwar/tree/main/sei).

**Comment by LornaSIL** [comment]
>Now running the MLP scripts, the only one missing is "suz-Sunu" (https://github.com/sillsdev/mui-language-picker/issues/28).

I added the font, but I don't know where else a change needs to be made.

This does return the font: https://lff.api.languagetechnology.org/family/seisunuwar
But, this does not return anything: https://lff.api.languagetechnology.org/lang/suz-Sunu

**Comment by DavidLRowe** [comment]
I see that you:
- added seisunuwar to families.json in silnrsi/fonts repo
- added seisunuwar to fallback.json in silnrsi/langfontfinder

@mhosken or @tim-eves Is there something we're missing?

**Comment by tim-eves** [comment]
Those are both present on the staging LFF endpoint:

- https://lff.languagetechnology.org/lang/suz-Sunu
- https://lff.languagetechnology.org/family/seisunuwar
> [!WARNING]
> **Do not use the staging endpoint in user facing code.**
> It is intended for testing only

The font family is on production because that got merged with fixes to the service code, but the lang data was checked into the main branch, and has not been merged into release yet.
About a minute after main gets merged into release it will show up
on the production endpoint https://lff.api.languagetechnology.org/

**Comment by DavidLRowe** [comment]
@LornaSIL Are we happy to move the seisunuwar addition to production?

**Comment by LornaSIL** [comment]
I don't know how to do that, but yes.

**Comment by DavidLRowe** [comment]
In a local clone of the repo I used:
```
git switch release
git merge main
git push
git switch main

```

and it seems to have successfully updated the release so that `https://lff.api.languagetechnology.org/lang/suz-Sunu` returns info for `seisunuwar`.

**Comment by tim-eves** [comment]
> I don't know how to do that, but yes.

I addition to the procedure that @DavidLRowe explained, you can also create a pull request, and merge that, or request someone else does review and merging if you feel that's needed. The potential advantage there is doing it all through GitHub's UI.

**Comment by jvgaultney** [comment]
Maybe document this in the Chord workflow docs?
